### ISSUE TITLE
[Request]: Trader's Little Helper

### ISSUE THREAD

**Issue by milnak** [ScoopInstaller/Extras#10482]
### Prerequisites

- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.

### Criteria

- [X] Reasonably well-known and widely used (e.g. if it's a GitHub project, it should have at least 100 stars and/or 50 forks)
- [X] English interface (or at least English documentation)
- [X] Latest stable version
- [X] Full version (i.e. not a trial version)
- [X] Fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Name

Trader's Little Helper

### Description

Encoding/Decoding/Verification of audio formats

### Homepage

https://tlh.easytree.org/

### Download Link(s)

https://tlh.easytree.org/files/tralih284185.exe

### Some Indication of Popularity/Repute

Very commonly used app in online live music trader scene.

**Comment by Zliced13** [comment]
I don't think this should be added to Scoop, it's got pretty high positives on online virus scanners.

https://www.virustotal.com/gui/file/24b48cabc84e5df3e3ce06f82e16a9237cd562519b36ce4425455e6d379c8c8e?nocache=1
https://metadefender.opswat.com/results/file/bzIzMDIyMEJmWEtUUk5yMEdSSTN2ektXSWg/regular/overview
https://virusscan.jotti.org/en-US/filescanjob/h4p5qqe1s5

**Comment by milnak** [comment]
"Pretty high positives"?  Let's see: "Antiy-AVL", "CrowdStrike Falcon", "Trapmine"?  Who has heard of these three?  

In any case, I have used this app for years, have spoken to the author, and have never seen any real world reports of virus or malware with this software.  I'm happy to put you in touch with the author.

**Comment by Zliced13** [comment]
Ok, I'll see what I can do.
